### PR TITLE
Add README.md for pointer authentication test

### PR DESCRIPTION
**Pull request by mmxsrup**
I added README.md in `examples/sw/simple_system/pa_test`.I described how to build the simulator and software, how to run the simulator.  Please tell me if my English is wrong.

### CODE REVIEW

File: examples/sw/simple_system/pa_test/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Ibex Pointer Authentication Test is an application to test pac/aut instructions
+added for pointer authentication. This test confirms that pac/aut instructions can be executed correctly,
+and an exception occurs using pre-authentication pointers. 
+We are using Ibex Simple System, so see `examples/simple_system/README.md` for more details.
+
+## Prerequisites
+
+Basically, you need the prerequisites written `examples/simple_system/README.md`.
+However, please use the rv32imcb Tool Versions for RISC-V Compiler Toolchain.
+This test uses the 4-operand version of the instruction (this format is defined in bitmanip extension),
```
Comment (vogelpi) [inline examples/sw/simple_system/pa_test/README.md:12]: "The aut instruction executed in this test uses the R4-type format (three source registers rs1, rs2, rs3 and one destination register rd). This format is only used by floating-point fused multiply-add (F) and some bit-manipulation (B) instructions".

File: examples/sw/simple_system/pa_test/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+so you have to use a compiler that supports the bitmanip extension.
+
```
[the remark this answers is not in the document]
Comment (mmxsrup) [reply examples/sw/simple_system/pa_test/README.md:23]: Thank you! Done here.

File: examples/sw/simple_system/pa_test/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+#  Ibex Pointer Authentication Test
+
+Ibex Pointer Authentication Test is an application to test pac/aut instructions
+added for pointer authentication. This test confirms that pac/aut instructions can be executed correctly,
+and an exception occurs using pre-authentication pointers.
```
[the remark this answers is not in the document]
Comment (mmxsrup) [reply examples/sw/simple_system/pa_test/README.md]: Done here. Thank you!

File: examples/sw/simple_system/pa_test/README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+The Simple System simulator binary can be built via FuseSoC. From the Ibex repository root run:
+
+```
+fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ibex:ibex_simple_system --RV32M=1 --RV32E=0 --PMPEnable=1 --PointerAuthentication=1
+```
+
+## Building Software
+
+To build the pointer authentication example, from the Ibex reposistory root run:
```
[the remark this answers is not in the document]
Comment (mmxsrup) [reply examples/sw/simple_system/pa_test/README.md]: Done here. Thank you!

File: examples/sw/simple_system/pa_test/README.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+Ibex Pointer Authentication Test is an application to test pac/aut instructions
+added for pointer authentication. This test confirms that pac/aut instructions can be executed correctly,
+and an exception occurs using pre-authentication pointers. 
+We are using Ibex Simple System, so see `examples/simple_system/README.md` for more details.
+
+## Prerequisites
+
+Basically, you need the prerequisites written `examples/simple_system/README.md`.
+However, please use the rv32imcb Tool Versions for RISC-V Compiler Toolchain.
```
[the remark this answers is not in the document]
Comment (mmxsrup) [reply examples/sw/simple_system/pa_test/README.md]: Done here. Thank you!

File: examples/sw/simple_system/pa_test/README.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+AUT Instructions:           1
+```
+
+The simulator produces several output files
+
+* `ibex_simple_system.log` - The ASCII output written via the output peripheral
+* `ibex_simple_system_pcount.csv` - A csv of the performance counters
+* `trace_core_00000000.log` - An instruction trace of execution
+
+By see `ibex_simple_system.log`, you can confirm that the exception is occurring:
```
[the remark this answers is not in the document]
Comment (mmxsrup) [reply examples/sw/simple_system/pa_test/README.md]: Thank you for rewriting.

File: examples/sw/simple_system/pa_test/README.md (hunk 1 above)
Comment (mmxsrup) [reply examples/sw/simple_system/pa_test/README.md]: I added this description. Thank you!

**Comment by vogelpi** [comment]
Thanks @mmxsrup it looks mostly good now, but the line wrapping seems awkward. Sometimes the wrapping occurs much before 80 chars but still in the middle of the sentence. Can you please correct this?

**Pull request merged**